sky/utils/kubernetes_utils.py

```diff
@@ -148,6 +161,10 @@ def get_label_key(cls) -> str:
     def get_label_value(cls, accelerator: str) -> str:
         return get_gke_accelerator_name(accelerator)
 
+    @classmethod
+    def get_accelerator_from_label_value(cls, value: str) -> str:
+        return value.split('-')[-1].upper()
```

Comment:
accelerators may have hyphens in their names (e.g.,`nvidia-a100-80gb` would end up being parsed as `80GB` here).

Can we do parsing by lstripping `nvidia-` and/or `nvidia-tesla-`? (see possible names [here](https://cloud.google.com/kubernetes-engine/docs/how-to/gpus)).

Reply:
Good catch, will update.

Reply:
Done in https://github.com/skypilot-org/skypilot/pull/2638/commits/fa0df6ac83a2cdb8b50e6e6c78f472d3e2d70a4f